ooni/nettest.py

```diff
@@ -567,12 +566,18 @@ def makeMeasurement(self, test_instance, test_method, test_input=None):
         return measurement
 
     @defer.inlineCallbacks
-    def initializeInputProcessor(self):
-        for test_class, test_method in self.testCases:
+    def initialize(self):
+        for test_class, _ in self.testCases:
+            # Initialize Input Processor
             test_class.inputs = yield defer.maybeDeferred(
                 test_class().getInputProcessor
             )
 
+            # Run the setupClass method
+            yield defer.maybeDeferred(
+                test_class.setUpClass
```

Comment:
Why a few lines above it's `test_class().getInputProcessor` and here it's `test_class.setUpClass` (i.e. why in the former case `test_class` is invoked and in the latter not)?